index.html

```diff
@@ -4778,41 +4789,28 @@ <h2 id="AudioBufferSourceNode">
           </dt>
           <dd>
             <p>
-              An optional value in seconds where looping should begin if the
-              <code>loop</code> attribute is true. Its default
+              An optional <a>playhead position</a> where looping should begin
```

Comment:
We don't need to multiply by the sample rate anywhere anymore, right?  We can just delete that part about multiplication and don't need this at all.  I just find suddenly discussing playhead position in the intro to the node odd.